scripts/merge_repos_and_check_duplicates.py

```
@@ -1,97 +1,118 @@
 import sys
-import collections
-
 from re import search
 
-if len(sys.argv) < 2:
+
+if len(sys.argv) < 4:
     print(
-        'usage: merge_repos_and_check_duplicates.py [geocint-runner] [geocint-openstreetmap] [your private repo]'
+        "usage: merge_repos_and_check_duplicates.py "
+        "[geocint-runner] [geocint-openstreetmap] [your private repo]"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)